pom.xml

```diff
@@ -7,7 +7,27 @@
   <groupId>io.dapr</groupId>
   <artifactId>dapr-sdk-parent</artifactId>
   <packaging>pom</packaging>
-  <version>1.9.0-SNAPSHOT</version>
+    <dependencies>
```

Comment:
Do these need to be in the parent POM? They're only needed for the workflow project, right?